### DISCUSSION TITLE
Trash button not displayed

### DISCUSSION THREAD

**Discussion by metaveersee** [victornpb/undiscord#682] [4. Bugs and Issues]
Good evening, when i want to delete my messages, i launch Discord on my Edge browser (since it doesn't work on Google) but it doesnt work any more despite uninstalling everything and putting it back.

Do you have a solution ? Thanks in advance !
![image](https://github.com/user-attachments/assets/c2b98e94-8aea-4516-acb0-616e8858b48c)

**Comment by broccolicode** [comment]
Same for me but with Chrome. Was just using it without any issue, restarted, and now the trash can is gone.

**Reply by bekkibau** [reply]
Fixed this issue on my fork for Chrome if you still need https://github.com/bekkibau/deletecord

**Comment by Erilsio** [comment]
Trash button doesn't show up for me either on Opera GX
